### ISSUE TITLE
Applying ENCODE E2G to different cell types

### ISSUE THREAD

**Issue by mbond0718** [karbalayghareh/ENCODE-E2G#1]
Hello! 

I just read the new preprint and am very excited to apply this new and improved version to my own data. If I am understanding the code correctly, it looks like you need to provide some sort of predicted enhancer-gene pairs that are already annotated with ABC scores and other epigenetic information (the CRISPRi file). Is this interpretation correct? And if so, would I be able to generate my own dataset that is formatted the same as the CRISPRi file with my own data?

Basically I have matching RNA-seq, H3K27ac ChIP, ATAC-seq, and Hi-C data and I would love to apply the E2G model on it, but I was wondering if you had any tips/suggestions on how to generate the file that I described above. 

Thank you in advance!

Marielle
